src/components/ErrorFallback/ErrorFallback.tsx

```
@@ -10,6 +10,9 @@ export const ErrorFallback = ({
   error: Error;
   resetErrorBoundary: () => void;
 }) => {
+  // eslint-disable-next-line no-console
+  console.error(error);
```

This looks like debug code, it's not part of any A/Cs

---

The `ErrorBoundary` component catches and hides all errors, making it impossible to understand what happened. So this will help us when debugging the issue.